Integrations/integration-ReversingLabs_A1000.yml

```diff
@@ -0,0 +1,452 @@
+description: ReversingLabs A1000 Malware Analysis Platform
+detaileddescription: |-
+  To generate a token for a given username you can use a simple POST request with the user credentials.
+  ```
+  curl -X POST --form "username=your-username" --form 'password=your-password' https://a1000-server-name-or-ip/api-token-auth/
+  ```
+releaseNotes: Integrate with A1000 to analyse samples
+configuration:
+- display: Base URL of the A1000 box
+  name: base
+  defaultvalue: https://a1000.reversinglabs.com
+  type: 0
+  required: true
+- display: Token to use for authentication
+  name: token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Provide extended information when available
+  name: extended
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |
+    import requests
+    from requests.auth import HTTPBasicAuth
+    import re
+    import os
+    import shutil
+    from zipfile import ZipFile
+
+    base_url = demisto.getParam('base')
+    if base_url[-1] == '/':
+        base_url = base_url[0:-1]
+    token = demisto.getParam('token')
+    headers = {'Authorization': 'Token ' + token}
+    extended = demisto.getParam('extended')
```

Comment:
base_url => BASE_URL
token => TOKEN
headers => HEADERS
extended => EXTENDED
we use upper case for global vars